app/controllers/provider/admin_controller.rb

```diff
@@ -26,7 +26,7 @@ def start_onboarding_wizard
   end
 
   def onboarding_active?
```

Comment:
I think that we should remove this method too. Since the concept of 'active?' doesn't exist anymore. We should just use onboarding.persisted? instead